dbos-config.schema.json

```diff
@@ -14,43 +14,43 @@
       "enum": ["node", "python"]
     },
     "database_url": {
-      "type": ["string", "null"],
+      "type": "string",
```

Comment:
What is the need for these changes? If I understand correctly, this means that if you set the value to an unset environment variable, validation will fail instead of falling back to the default.

Reply:
`substituteEnvVars` sets missing env values to an empty string, not null. but I'll add a test